src/components/Channels/index.tsx

```diff
@@ -0,0 +1,51 @@
+import "./styles.scss"
+import { StrictMode } from "react";
+import Toggle from './toggle';
+import IconButton from '@mui/material/IconButton';
```

Comment:
why do you have Toggle the component from toggle.tsx file, haven't we agreed that every component should have it's own folder?